deps/rabbit/src/gm.erl

```diff
@@ -1373,11 +1373,12 @@ with_member_acc(Fun, Id, {MembersState, Acc}) ->
     {MemberState, Acc1} = Fun(find_member_or_blank(Id, MembersState), Acc),
     {store_member(Id, MemberState, MembersState), Acc1}.
 
-find_member_or_blank(Id, MembersState) ->
+find_member_or_blank(Id, MembersState) when is_map(MembersState) ->
     case maps:find(Id, MembersState) of
         {ok, Result} -> Result;
         error        -> blank_member()
-    end.
+    end;
+find_member_or_blank(_Id, _MembersState) -> blank_member().
```

Comment:
I would match on `undefined` for the member state. It's generally better to match on what we expect rather than have a catch-all clause that can unexpectedly drop valid arguments we want to know about.

But since `gm` is going away I don't have strong feelings about this, either way is fine.